test/unit/app/routes/endemics/species-numbers.test.js

```diff
@@ -0,0 +1,104 @@
+const cheerio = require('cheerio')
```

Comment:
Can you also add tests for the eligible and ineligible pages? Sonar Cloud is flagging them as uncovered